src/pages/portfolio-page/index.tsx

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { Container, Box, makeStyles, Card, useTheme, useMediaQuery } from '@material-ui/core';
+
+import { DetailsCard } from '../../components/DetailsCard';
+import { SectionTitle } from '../../components/SectionTitle';
+import { PortfolioCard } from '../../components/PortfolioCard';
+
+import { projectData, userData } from './data';
+
+const useStyles = makeStyles((theme) => ({
+  container: {
+    padding: 0,
+    [theme.breakpoints.up('lg')]: {
+      display: 'flex',
+      minHeight: '100vh',
+      padding: theme.spacing(8, 2, 0, 2),
+    },
+  },
+  aside: {
+    display: 'none',
+    [theme.breakpoints.up('lg')]: {
+      display: 'block',
+      position: 'sticky',
+      top: theme.spacing(8),
+      height: 668,
+      width: 280,
+      marginRight: theme.spacing(4),
+    },
+  },
+  main: {
+    [theme.breakpoints.up('lg')]: {
+      width: '100%',
+    },
+  },
+  mainContent: {
+    backgroundColor: 'transparent',
+    borderRadius: 16,
+    margin: theme.spacing(0, 3, 7, 3),
+
+    [theme.breakpoints.up('sm')]: {
+      backgroundColor: theme.palette.background.paper,
+    },
+
+    [theme.breakpoints.up('lg')]: {
+      display: 'flex',
+      flexDirection: 'column',
+      margin: theme.spacing(0, 0, 7, 0),
+      boxShadow: '0 40px 50px 0 rgba(103, 118, 128, 0.1)',
```

Comment:
Put this shadow to some variable, as you're using it multiple times

Reply:
Will do 👍